twitter_cldr.js

```diff
@@ -2,8 +2,18 @@
 // http://www.apache.org/licenses/LICENSE-2.0
 
 exports.load = function (locale) {
-  var TwitterCLDR = require('./full/core');
-  var TwitterCLDRDataBundle = require('./full/' + locale);
-  TwitterCLDR.set_data(TwitterCLDRDataBundle);
-  return TwitterCLDR;
+  /**
+   * Generated programmatically via Ruby:
+   *
+   * ```rb
+   * Dir.entries('./full').filter{ |x| x.end_with?('.js') && x != 'core.js' }.map{ |x| x.slice(0, x.length - 3) }.sort
```

Comment:
Not to be "that guy" but this could be a little more idiomatic:

```suggestion
   * Dir.entries('./full').select { |x| x.end_with?('.js') && x != 'core.js' }.map { |x| x[0..-4] }.sort
```